nvbios/nvbios.c

```diff
@@ -709,8 +709,26 @@ void printscript (uint16_t soff) {
 				soff += 7;
 				break;
 			case 0x9e:
+				/*
```

Comment:
None of the other ops have docs ... Is the right place to be sticking in docs?

Reply:
TBH I think the more docs the better.

Reply:
Then you end up with 30 copies of them, all slightly different. No. One place, and this isn't it.

Reply:
Hi @imirkin - I'm preparing for the Ampere devinit patches, and am re-scrubbing this for merging to start from a clean base. On the documentation, if not this location, then which file should I add them in the envytools repo?